tests/serializers/test_typed_dict.py

```diff
@@ -8,14 +8,15 @@
 from pydantic_core import SchemaSerializer, core_schema
 
 
-def test_typed_dict():
+@pytest.mark.parametrize('extra_behavior_kw', [{}, {'extra_behavior': 'ignore'}, {'extra_behavior': None}])
+def test_typed_dict(extra_behavior_kw: Dict[str, Any]):
     v = SchemaSerializer(
         core_schema.typed_dict_schema(
             {
                 'foo': core_schema.typed_dict_field(core_schema.int_schema()),
                 'bar': core_schema.typed_dict_field(core_schema.bytes_schema()),
             },
-            extra_behavior='ignore',  # this is the default
+            **extra_behavior_kw,
```

Comment:
the `None` case will be filtered out by `typed_dict_schema`, if you pass it directly, i think it will cause an error.

Reply:
changed